packages/angular/test/base/e2e/src/router-link.spec.ts

```diff
@@ -138,6 +138,7 @@ function testForward() {
   cy.get('app-router-link-page #canGoBack').should('have.text', 'true');
 
   cy.go('back');
+  cy.wait(500);
```

Comment:
Are the timeout changes here necessary? If not, could we revert them? (Sorry meant to comment on this with my last round of reviews, but it slipped my mind)

Reply:
Also if possible, could you allow me to push changes to your branch? I can resolve the merge conflicts for you.

Reply:
I think, it is necessary. Before, the back transition was instant and now it takes the time for the animation which is why we need the additional `wait`
I invited you to my fork. Thanks for taking care of the merge conflicts.